### PR TITLE
Pr

### PR DESCRIPTION
**Pull request by YehorSilahin96**
Added constant patterns

### CODE REVIEW

File: src/com/company/Main.java [hunk 1]
```diff
@@ -4,37 +4,40 @@
...
         System.out.print("Enter email: ");
         String mail = in.nextLine();
 
-        boolean res1 = name.matches("^([A-Z][a-z]*((\\s)))+[A-Z][a-z]*$"); // Always put strings like this to constants. Do you know why? 
+        boolean res1 = name.matches(PATTERN_NAME);
         if (res1) {
             System.out.println("Name validation successful");
-        }
-            else { // please, fix formatting. 'else' must be on the same line with '}' 
+            }
```
Comment (AnnaDremluga) [inline src/com/company/Main.java:24]: I don't see changes

Comment (AnnaDremluga) [inline src/com/company/Main.java:24]: I don't see changes

File: src/com/company/Main.java [hunk 2]
```diff
@@ -4,37 +4,40 @@
...
+            else {
             System.out.println("Illegal. Name must contains only A-Z, a-z and space");
-            } // please, fix indentation
+            }
```
Comment (AnnaDremluga) [inline src/com/company/Main.java:27]: I don't see changes

File: src/com/company/Main.java [hunk 3]
```diff
@@ -4,37 +4,40 @@
...
         System.out.print("Enter name and last name: ");
         String name = in.nextLine();
         System.out.print("Enter phone number: ");
-        String num = in.nextLine();
+        String number = in.nextLine();
         System.out.print("Enter email: ");
         String mail = in.nextLine();
 
-        boolean res1 = name.matches("^([A-Z][a-z]*((\\s)))+[A-Z][a-z]*$"); // Always put strings like this to constants. Do you know why? 
+        boolean res1 = name.matches(PATTERN_NAME);
```
Comment (AnnaDremluga) [inline src/com/company/Main.java:21]: name 'res1' does not tell us what is it about.

File: src/com/company/Main.java (hunk 1 above)
Comment (YehorSilahin96) [reply src/com/company/Main.java:24]: Done

File: src/com/company/Main.java (hunk 2 above)
Comment (YehorSilahin96) [reply src/com/company/Main.java:27]: DOne

Comment (YehorSilahin96) [reply src/com/company/Main.java:27]: Done

File: src/com/company/Main.java (hunk 3 above)
Comment (YehorSilahin96) [reply src/com/company/Main.java:21]: Done

**Pull request merged**